quick_purchase/views/purchase_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+
+        <record id="view_purchase_form" model="ir.ui.view">
+            <field name="model">purchase.order</field>
+            <field name="inherit_id" ref="purchase.purchase_order_form" />
+            <field name="arch" type="xml">
+                <field name="order_line" position="before">
+                    <button name="add_product"
+                            string="Add product"
+                            type="object"
+                            attrs="{'invisible': [('state', 'not in', ('draft', 'sent'))]}"
+                            class="oe_highlight"/>
+                </field>
+                <field name="order_line" position="attributes">
+                    <attribute name="create">0</attribute>
```

Comment:
Hi @mourad-ehm,
Why do you set order_line create to 0 ?
regards.

Reply:
It was one of my question to. The idea was to lead the user to use the add button instead.

I'm not sure what is the better best behaviour ?

Maybe we could request ourself and merge as it. We could add another way in v10, if you think, we can let create with transitional way

Reply:
> It was one of my question to. The idea was to lead the user to use the add button instead.

I think we should not.
* To add a new line with a product, user should have the possibility to use the classical way, or the new way introduced by this module. The new button is interesting only for mass editing product / quantity.
* We loose the possibility to add a line without product. Don't we ? If yes, it is not possible.

regards.

Reply:
you're right. Ok for that